### PR TITLE
Adds extension to Array, simplifies SQLiteDatabase.Version.init(_:)

### PR DESCRIPTION
**Pull request by valeriyvan**

### CODE REVIEW

File: Common/Database/SQLite/SQLiteDatabase.swift [hunk 1]
```diff
@@ -157,14 +157,10 @@ public class SQLiteDatabase: Database, Assertable {
...
-            var parts = str.components(separatedBy: ".")
-            if parts.count == 3 {
-                (major, minor, patch) = (parts[0], parts[1], parts[2])
-            } else if parts.count == 2 {
-                (major, minor, patch) = (parts[0], parts[1], "")
-            } else {
-                (major, minor, patch) = (str, "", "")
-            }
+            let parts = str.components(separatedBy: ".")
+            major = parts[safe:0] ?? ""
```
Comment (DmitryBespalov) [inline Common/Database/SQLite/SQLiteDatabase.swift]: 😍 please add space after `:` as per project code style.

We can go even further:

```
public extension Array where Element == String {

  subscript(stringAt index: Int) -> Element {
     return self[safe: index] ?? ""
  }

}
```

File: Common/Common/FoundationExtensions.swift [hunk 2]
```diff
@@ -32,6 +32,14 @@ public extension String {
 
 }
 
+public extension Array {
+
+    subscript (safe index: Int) -> Element? {
```
Comment (DmitryBespalov) [inline Common/Common/FoundationExtensions.swift]: please no space between method name and arguments list -> `subscript(safe`

Comment (valeriyvan) [reply Common/Common/FoundationExtensions.swift]: removed

File: Common/Database/SQLite/SQLiteDatabase.swift (hunk 1 above)
Comment (valeriyvan) [reply Common/Database/SQLite/SQLiteDatabase.swift]: Added spaces

**Review by DmitryBespalov — APPROVED** [review]

**Pull request merged**